### ISSUE TITLE
resmgr regression

### ISSUE THREAD

**Issue by bertsky** [OCR-D/core#908]
With v2.38 I now get:
```
INFO ocrd.resource_manager - Inspecting 'ocrd-anybaseocr-dewarp --dump-json' for resources
  File "ocrd/cli/resmgr.py", line 126, in download
    if resdict['url'] == '???':
TypeError: list indices must be integers or slices, not str
```

The reason is this line:

https://github.com/OCR-D/core/blob/ceb9992ab113f1b3933b2af33819ceeef0f085b7/ocrd/ocrd/resource_manager.py#L123

Notice the empty list instead of empty dict.

**Comment by bertsky** [comment]
Wait, seems like this _should_ be a list. So the error is on `cli.resmgr`'s part...

**Comment by kba** [comment]
Yes, sorry, this was fixed as part of #904, I've merged your changes into it.

**Comment by bertsky** [comment]
Oh, sry, already forgot about that one. Thx!
